integreat_cms/cms/templates/_base.html

```diff
@@ -211,12 +196,19 @@
                         {% translate "Imprint" %}
                     </a>
                 {% endif %}
-                {% if perms.cms.view_user or perms.cms.view_languagetreenode %}
+                {% if perms.cms.manage_translations or perms.cms.view_user or perms.cms.view_languagetreenode %}
                     <!-- Settings Section-->
```

Comment:
As @timoludwig suggest in the issue report I think we should also check if at least one MT provider has been enabled in this region. Therefore I think we should add the check "if DEEPL_ENABLED"

Reply:
I'd show the entry when at least either DeepL or SUMM.AI are enabled.

Reply:
Oh yes, of course. I forgot about SUMM.AI for a second :see_no_evil:

Reply:
Sorry, I overlooked it 🙈 
Now MT providers are checked too.